contracts/src/v0.8/keystone/interfaces/ICapabilityConfiguration.sol

```diff
@@ -31,4 +31,6 @@ interface ICapabilityConfiguration {
     uint64 donCapabilityConfigCount,
     uint32 donId
   ) external;
+
+  function getNextDONId() external view returns (uint32);
```

Comment:
Let's just have an entirely separate interface for our contract to use, we should avoid changing existing interfaces or contracts:

```
interface ICapabilitiesRegistry {
    function getNextDONId() external view returns (uint32);
}
```

Reply:
We could even just define this interface in the DonId claimer file.

Reply:
https://github.com/smartcontractkit/chainlink/pull/16926/commits/e5e4a9ed063b98f455d7792e02a2adf5da36ca33